packages/xo-web/src/common/sorted-table/index.js

```diff
@@ -1006,4 +1006,5 @@ class SortedTable extends Component {
   }
 }
 
-export default withRouter(SortedTable)
+// withRouter is used to force the render on route change by injecting the route properties in props
```

Comment:
@Rajaa-BARHTAOUI 

```suggestion
// `SortedTable` needs to change the root on sorting/filtering items, storing the route in the `contextType` will not trigger a render on the route change
```

Reply:
What is this? Did you want to change it?

Reply:
After talking with @Rajaa-BARHTAOUI , we saw that the precedent comment doesn't describe the problematic. So i suggested a new comment.

Reply:
...when sorting/filtering items ?
You can separate it in two sentences.

Reply:
Why?

Reply:
```suggestion
// withRouter is needed so that the SortedTable does trigger a render when sorting/filtering items.
```

Reply:
What's about ?
```suggestion
withRouter is needed to trigger a render on filtering/sorting items
```